src/libraries/System.Private.Uri/src/System/Uri.cs

```diff
@@ -399,7 +399,7 @@ public Uri(Uri baseUri!!, string? relativeUri, bool dontEscape)
         //
         // Uri(string, UriKind);
         //
-        public Uri(string uriString!!, UriKind uriKind)
+        public Uri([StringSyntax(StringSyntaxAttribute.Uri)] string uriString!!, UriKind uriKind)
```

Comment:
https://github.com/dotnet/runtime/pull/67621/files#diff-947517a24f24fb439159a5c8f47ad9bde9a8c48e5f3ab9672a4fff7a8a73f45cR251 uses `[StringSyntax(StringSyntaxAttribute.Uri, "uriKind")]`.  Assuming the uriKind is relevant to the syntax highlighters/etc, shouldn't the same be done here?

Reply:
Yup, good catch.